pytests/test_errors.py

```diff
@@ -112,7 +112,7 @@ def testJSStackOverflowOnLoading(env):
 test(1);
 redis.register_function('test', test);
     """
-    env.expect('CONFIG', 'SET', 'redisgears_2.lock-redis-timeout', '10000')
+    env.expect('CONFIG', 'SET', 'redisgears_2.lock-redis-timeout', '50000')
```

Comment:
I see that we already set a 10 seconds timeout here? So maybe I was wrong and its not timeout?

Reply:
Ohh I believe the failure was on `testJSStackOverflow` no?
Let add timeout there?

Reply:
Ok so I have no idea why this is happening, 10 seconds should be more then enough